src/cli/ImportFeedCommand.ts

```diff
@@ -1,35 +1,30 @@
 import AdmZip = require("adm-zip");
-import {CLICommand} from "./CLICommand";
-import {FeedConfig} from "../../config";
-import {FeedFile} from "../feed/file/FeedFile";
-import {MySQLSchema} from "../database/MySQLSchema";
-import {DatabaseConnection} from "../database/DatabaseConnection";
+import fs = require("fs-extra");
+import LineByLineReader = require('line-by-line');
 import * as path from "path";
-import {MySQLTable} from "../database/MySQLTable";
 import * as memoize from "memoized-class-decorator";
-import fs = require("fs-extra");
-import {MultiRecordFile} from "../feed/file/MultiRecordFile";
-import {RecordWithManualIdentifier} from "../feed/record/FixedWidthRecord";
-import {MySQLStream, TableIndex} from "../database/MySQLStream";
-import byline = require("byline");
-import streamToPromise = require("stream-to-promise");
+import {FeedConfig} from "../../config";
+import {FeedFile, MultiRecordFile} from "../feed/file";
+import {RecordWithManualIdentifier} from "../feed/record";
+import {DatabaseConnection, MySQLSchema, MySQLStream, MySQLTable, TableIndex} from "../database";
+import {CLICommand} from "./CLICommand";
 
 const getExt = filename => path.extname(filename).slice(1).toUpperCase();
-const readFile = filename => byline.createStream(fs.createReadStream(filename, "utf8"));
 
 /**
  * Imports one of the feeds
  */
 export class ImportFeedCommand implements CLICommand {
 
-  private index: {[name: string]: MySQLTable} = {};
+  private index: { [name: string]: MySQLTable } = {};
```

Comment:
You can use Record<Key, Type> here:
`private index: Record<string, MySQLTable>;`

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype